overrides/config/ftbquests/quests/chapters/53659F546FCEEC84.snbt

```diff
@@ -52,13 +52,7 @@
 				"     Already, because of those eyes you've created, you have become Powerful enough to use complex storage, a slightly stronger material infused with the god Neptune's Essence, Neptunium and enter the Underworld! So please, for our sake, keep it coming. Take this offering, the only thing i can materialize into your world from here, as a sign of peace. And please don't forget: the more eyes you collect, the more Powerful you become..."
 			]
 			hide: true
-			icon: {
-				Count: 1b
-				ForgeCaps: {
-					"dungeons_libraries:built_in_enchantments": { }
-				}
-				id: "paraglider:heart_container"
-			}
+			icon: "paraglider:heart_container"

```

Comment:
any idea why these keep switching every once in a while?

Reply:
nope lol, I dont even know what it is

Reply:
actually, it might be in the dialog section/but I never go in there so idk

Reply:
not a big deal but in the future you can check what files where edited with `git status` or better go to the "source control" tab in vs code and compare changes. for stuff like this you could unstage the file if the change isn't relevant to the commit

it might keep switching back and forth as ftb regenerates the files but if it works either way then we don't need to keep committing them every time

Reply:
also I just mean it switching "icon: " between just a string an a larger object with data that doesn't seem to even be used

Reply:
![image](https://github.com/chocolate-edition/Chocolate-Edition/assets/66807613/864a8d08-3145-4bdf-82eb-2dfd15202410)
 here?

Reply:
yeah! its basically the same as `git status` but with a better interface for comparing changes in individual files and individually staging instead of doing `git add .` before you commit

Reply:
kk ill try it next time I push!